src/index.html

```diff
@@ -11,6 +11,37 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="post"
+      data-qa="big"
+    >
+      <img 
+      src="images/Search.svg" 
+      alt="search"
+      class="icon-big"
+    >
```

Comment:
```suggestion
      <img 
        src="images/Search.svg" 
        alt="search"
        class="icon-big"
      >
```
![image](https://user-images.githubusercontent.com/96409563/217919386-03e0b2d9-d2be-4b80-960f-0615e745e5ec.png)
Fix all cases